scripts/exglobal_atmos_analysis.sh

```diff
@@ -866,7 +866,6 @@ OBS_INPUT::
    iasibufr       iasi        metop-c     iasi_metop-c        0.0     1     1
    sstviirs       viirs-m     npp         viirs-m_npp         0.0     4     0
    sstviirs       viirs-m     j1          viirs-m_j1          0.0     4     0
-   abibufr        abi         g18         abi_g18             0.0     1     0
```

Comment:
Thanks for checking on this with other DA folks @RussTreadon-NOAA !

Reply:
Thanks @KateFriedman-NOAA  for noting this.  The failure is due to problems with the crtm coefficients.   Ideally the crtm team provides a fix instead of g-w sidestepping the problem.

Reply:
Ben Johnson comes to mind but I'm not sure if he is the right person to contact.   @ADCollard , @emilyhcliu , @HaixiaLiu-NOAA can probably point us at the right person.   Once we have the right coefficients, the library team needs to add the corrected files to `CRTM_FIX` on various machines.   Doing so on WCOSS2 will likely require an increment in the CRTM version number (NCO policy).   We need to decide if such a change is worth the ripples it creates.

Reply:
@RussTreadon-NOAA Gotcha, thanks for those details! Given the fact that the `crtm` version will likely increment to accommodate this fix we can plan to leave this line out while still using `crtm/2.4.0` and add it back in should we get a new `crtm/2.5.0.` Please advise otherwise though, thanks!

Reply:
The change is required given problems with the crtm coefficients.   We can't wait for the crtrm files to be fixed.   The danger in removing functionality is that (1) the reason for the removal may be forgotten, (2) a fix is not actively pursued.

Reply:
> @emilyhcliu , @ADCollard , @HaixiaLiu-NOAA. What ABI behavior do you want from the `develop` g-w? Retain, skip, or something else?

The abi_g18 coeff files are in the crtmv2.4.0 installation already, so we can keep the abi in the develop g-w now. Thanks.

Reply:
@HaixiaLiu-NOAA The `abi_g18.SpcCoeff.bin` file is not in the WCOSS2 production `crtm/2.4.0` install, nor the hpc-stack `crtm/2.4.0` installs. Hence this related global-workflow issue and why I deleted this line, see https://github.com/NOAA-EMC/global-workflow/issues/1321.

```
kate.friedman@dlogin01:~> ll /apps/ops/prod/libs/intel/19.1.3.304/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin
ls: cannot access '/apps/ops/prod/libs/intel/19.1.3.304/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin': No such file or directory
```
```
-bash-4.2$ ll /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin
ls: cannot access /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin: No such file or directory
-bash-4.2$ ll /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-2022.1.2/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin
ls: cannot access /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-2022.1.2/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin: No such file or directory
```

I do see it in the newer EPIC hpc-stack `crtm/2.4.0` installs though (good):
```
-bash-4.2$ ll /scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/intel-2022.1.2/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin
-rw-r--r-- 1 role.epic nems 872 Nov 10 19:50 /scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/intel-2022.1.2/crtm/2.4.0/fix/abi_g18.SpcCoeff.bin
```
...but we aren't yet using those stacks (I am currently testing them on Hera/Orion).

To be able to add this line back in we'd need:

1. The WCOSS2 `crtm/2.4.0` updated to add in the missing `abi_g18.SpcCoeff.bin` file.
2. Finish moving to the EPIC hpc-stacks elsewhere.

The EPIC hpc-stacks are intel 2022 but the GSI is still using intel 2018. I can force the workflow to load the EPIC hpc-stack intel 2022 `crtm/2.4.0` at runtime but the GSI would still be building with the pre-EPIC hpc-stack intel 2018 `crtm/2.4.0`. It should work but will continue the inconsistency between the workflow and GSI. The GSI needs to be updated to intel 2022 regardless (another issue).

As it stands right now, the `develop` branch will break for newer CDATEs with this line included and us using the `crtm/2.4.0` fix files as they are now. As observed in https://github.com/NOAA-EMC/global-workflow/issues/1321.

Please advise. Perhaps I can comment out this line for now with a note to uncomment when we move to `crtm/2.4.0` that includes the `abi_g18.SpcCoeff.bin` file everywhere? Or is there another way to temporarily disable this without removing the line? Thanks!

Reply:
@KateFriedman-NOAA You are correct. I checked the Hera EPIC hpc-stack installation and noticed that the abi_g18 coeff files exist there. The WCOSS2 crtmv2.4.0 installation needs an update to include N21 and G18 coeff files and is awaiting Andrew's approval. Andrew is currently testing the N21 coeff files and will let Hang Lei know if the N21 coeff files are good to go this week, at which point the WCOSS2 v2.4.0 can be updated to include these new coeff files.

Your suggestion of commenting out the abi_g18 for now sounds good to me.

Reply:
> I'll leave the abi_g18 line commented out for now.

@HaixiaLiu-NOAA Slight correction to my reply...I will leave the abi_g18 _removed_ (not commented out) for now, which mimics the change made in ops recently. I will add the line back in when it's added back in ops.

Reply:
Added note to https://github.com/NOAA-EMC/global-workflow/issues/1356 to add line back in when processing upcoming ops updates.